sktree/datasets/hyppo.py

```diff
@@ -47,14 +47,33 @@ def make_quadratic_classification(n_samples: int, n_features: int, noise=False,
     return x, v
 
 
+MARRON_WAND_SIMS = {
```

Comment:
Can you add a docstring above describing what the list is comprised of and what the simulations are?